src/poliastro/czml/extract_czml.py

```diff
@@ -276,7 +277,11 @@ def add_ground_station(
                 )  # get semi-major and semi-minor axises
             else:
                 a, b = Earth.R.to(u.m).value, Earth.R_polar.to(u.m).value
```

Comment:
While you're at it, we can do:

```suggestion
                a, b = Earth.R.to_value(u.m), Earth.R_polar.to_value(u.m)
```